packages/babel-plugin-apply-mdx-type-props/test/index.test.js

```diff
@@ -19,7 +19,7 @@ const transform = value => {
 describe('babel-plugin-add-mdx-type-prop', () => {
   test('should add `mdxType` to components', () => {
     expect(transform('var d = <div><Button /></div>').code).toEqual(
-      'var d = <div><Button mdxType="Button" /></div>;'
+      'var d = <div><Button parentName="div" mdxType="Button" /></div>;'
```

Comment:
could you expand on why `parentName` needs to be added?

Reply:
Because it’s already added: https://github.com/mdx-js/mdx/blob/3080444e2a40d79e3a3ac61ec1266154f95f77bf/packages/mdx/mdx-hast-to-jsx.js#L179

The reason is so that `ol.li` can be used as a key in `components`, in which case, when a `li` has a `parentName` of `ol`, that component is used

Reply:
with the move to `hast` + `estree` is it still needed?
Could adopters use https://github.com/syntax-tree/hast-util-select to have an even more flexible API?

Reply:
I'd like to see it removed. I'll open a new issue